src/js/helpers/general.js

```diff
@@ -14,6 +15,15 @@ function setTuningData(tuning) {
   jQuery('#txt_tuning_data').val(tuning)
 }
 
+function isTuningDataAvailable(alertIfNot = false, alertMsg = 'No tuning data.') {
```

Comment:
I would split this function into 2. There should be a separate function, which's job is only to do the check from line 20, so that if we later have a better definition on what counts as an empty tuning data apart from an empty string, then we can refine it there.